trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/fuselage_correction.py

```diff
@@ -41,7 +41,7 @@ def fuselage_correction(state,settings,geometry):
     wings_lift_comp = state.conditions.aerodynamics.lift_coefficient
     
     # total lift, accounting one fuselage
-    aircraft_lift_total = wings_lift_comp * fus_correction 
+    aircraft_lift_total = wings_lift_comp #* fus_correction 
```

Comment:
It would be better to set this to 1 or just remove the functionality.

Reply:
I had been using the VLM including the fuselage before, but when I merged with develop that functionality was removed. So I put that back to normal. If we include the fuselage, the coefficient should go to 1.